Maths/test/ParityOutlier.test.js

```diff
@@ -9,17 +9,14 @@ describe('Testing parityOutlier function', () => {
     expect(parityOutlier([177, 5, 76, 1919])).toBe(76)
   })
 
-  it('should, if the given array has only one integer element, return the integer itself', () => {
```

Comment:
contradicts the algorithm, if there is only one even or odd number the return is null